simln-lib/README.md

```diff
@@ -0,0 +1,13 @@
+# sim-ln
+
+A Lightning Network simulation library that enables testing and analysis of payment routing behavior in a controlled environment.
+
+## Overview
+
+sim-ln provides a framework for simulating Lightning Network payment routing behavior, allowing developers and researchers to:
+
+- Create simulated Lightning Network topologies
+- Test payment routing strategies
+- Analyze network behavior under different conditions
+- Simulate real Lightning node implementations (LND, c-lightning, Eclair)
```

Comment:
I think we can remove this last line until we have the ability to run with simulated mode fully in simln